github/repos_projects.go

```diff
@@ -32,7 +36,7 @@ func (p Project) String() string {
 
 // ListProjects lists the projects for a repo.
 //
-// GitHub API docs: https://developer.github.com/v3/repos/projects/#list-projects
+// GitHub API docs: https://developer.github.com/v3/projects/#list-repository-projects
 func (s *RepositoriesService) ListProjects(owner, repo string, opt *ListOptions) ([]*Project, *Response, error) {
```

Comment:
This method should stay with `RepositoriesService` because it's still under `/repos/`.